ros_cross_compile/ros_cross_compile.py

```diff
@@ -134,6 +138,7 @@ def parse_args(args: List[str]) -> argparse.Namespace:
 
 def cross_compile_pipeline(
     args: argparse.Namespace,
+    data_collector: DataCollector = None,
```

Comment:
if this can be None, it should be `Optional[DataCollector]` - however, this should just be mandatory